src/Endpoints/class-analytics-post-detail-api-proxy.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Endpoints: Parse.ly `/analytics/post/detail` API proxy endpoint class
+ *
+ * @package Parsely
+ * @since   3.6.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\Endpoints;
+
+use stdClass;
+use WP_REST_Request;
+
+/**
+ * Configures the `/analytics/post/detail` REST API endpoint.
+ */
+final class Analytics_Post_Detail_API_Proxy extends Base_API_Proxy {
+
+	/**
+	 * Registers the endpoint's WP REST route.
+	 */
+	public function run(): void {
+		$this->register_endpoint( '/analytics/post/detail' );
+	}
+
+	/**
+	 * Cached "proxy" to the Parse.ly `/analytics/post/detail` API endpoint.
+	 *
+	 * @param WP_REST_Request $request The request object.
+	 */
+	public function get_items( WP_REST_Request $request ): stdClass {
+		return $this->get_data( $request );
+	}
+
+	/**
+	 * Generates the final data from the passed response.
+	 *
+	 * @param array<string, mixed> $response The response received by the proxy.
```

Comment:
Line 47 treats the items of the `$response` array as `stdClass` - is that right? If so, could `stdClass` be used here instead of `mixed`?

Reply:
Now tracked in #1209.